### PR TITLE
chapter 7

### PR DESCRIPTION
**Pull request by jabirG**

### CODE REVIEW

File: chapter-7.qmd [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+title: "Workflow: scripts and projects"
+---
+
+
+
+## Solutions
+
+1.what tip from twitter did i like?
+
+if you go to tools-global options - appearance you can change the background
```
Comment (jabirG) [inline chapter-7.qmd:11]:
```suggestion
If you go to tools-global options - appearance you can change the background.
```

File: chapter-7.qmd [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+title: "Workflow: scripts and projects"
+---
+
+
+
+## Solutions
+
+1.what tip from twitter did i like?
```
Comment (jabirG) [inline chapter-7.qmd:9]:
```suggestion
1. What tip from twitter did i like?
```

File: chapter-7.qmd [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+title: "Workflow: scripts and projects"
+---
+
+
+
+## Solutions
```
Comment (mine-cetinkaya-rundel) [inline chapter-7.qmd:7]: Remove this, since the whole thing is solutions.

File: chapter-7.qmd [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+1. What tip from twitter did i like?
+
+If you go to tools-global options - appearance you can change the background.
```
Comment (mine-cetinkaya-rundel) [inline chapter-7.qmd:11]:
```suggestion
If you go to Tools > Global options > Appearance, you can change the background.
```

Comment (mine-cetinkaya-rundel) [reply chapter-7.qmd:11]: I would add under which dropdown menu you can do that. And note that it's not just the background. You're changing the theme. So more accurate to say "you can change the theme".

**Review by mine-cetinkaya-rundel — CHANGES_REQUESTED** [review]
Second question is missing.

File: chapter-7.qmd [hunk 5]
```diff
@@ -0,0 +1,11 @@
+---
+title: "Workflow: scripts and projects"
+---
+
+
+
+## Solutions
+
+1. What tip from twitter did i like?
```
Comment (mine-cetinkaya-rundel) [inline chapter-7.qmd:9]: Remove the text here, just state the answer.

Comment (jabirG) [reply chapter-7.qmd:9]: i would think a future person would want to see the questions with the answers

**Pull request merged**